vendor/github.com/convox/stdsdk/client.go

```diff
@@ -407,6 +406,7 @@ func unmarshalReader(r io.ReadCloser, out interface{}) error {
 	}
 
 	data, err := ioutil.ReadAll(r)
+
```

Comment:
can you remove this file change? so it will not show if someone runs `go vendor mod`.